ci/scripts/utils/launch_java_agent.sh

```diff
@@ -46,6 +107,13 @@ export GH="${HOME}/bin/gh"
 command -v "${GH}"
```

Comment:
On hera, this causes a silent exit if `gh` is not in `${HOME}/bin`.  Suggest checking that it exists:
```suggestion
[[ -f "${GH}" ]] || echo "gh is not installed in ${HOME}/bin"
```